index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+</head>
+<body>
+<h1><b>About Me</b></h1>
+<p>My name is Aalitenaye. I am from Central Asia. I have a Bachelor of Business Administration Degree. I am 2 years old in Toronto.
+</p>
+<p>I envision to have my own business as long term goal, but my short term goal is to master web development skills, and excel in it.
+</p>
+<h2>My Interests;</h2>
+<ul>
+  <li>Programming
+  <li>Meditating
+  <li>Dancing
+</ul>
+<img src="images/programming.jpg" width="30%">
```

Comment:
These images don't show up, can you find out why and fix them? (Hint: capitalization matters)

Reply:
I worked on it, do I need to push it to github to see if the pictures show now?

Reply:
Yes, please add, commit and push your changes, and this pull request will be updated with your latest work.